torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -786,7 +785,6 @@ XLAGraphExecutor::ScheduleSyncTensorsGraph(
         TF_VLOG(3) << "Executing IR graph hash "
                    << torch::lazy::HashToString(hash)
                    << " on devices: " << absl::StrJoin(devices, ",");
-        // TODO(jwtan): Remove the WrapXlaData when inherits LazyGraphExecutor.
         results = WrapXlaData(xla::ComputationClient::Get()->ExecuteReplicated(
```

Comment:
can we still remove `WrapXlaData`?

Reply:
I guess we can once we restructure the xla::ComputationClient to inherit torch::lazy::ComputationClient, but that's later...